components/board/digital_interrupts_test.go

```diff
@@ -34,16 +34,21 @@ func TestBasicDigitalInterrupt1(t *testing.T) {
 	test.That(t, err, test.ShouldBeNil)
 	test.That(t, intVal, test.ShouldEqual, int64(2))
 
-	c := make(chan bool)
+	c := make(chan Tick)
 	i.AddCallback(c)
 
-	go func() { i.Tick(context.Background(), true, nowNanosTest()) }()
+	timeMicroSec := nowNanosTest()
```

Comment:
Should this function be renamed if we're doing everything in microseconds? I naively want to divide the righthand side by 1000 before storing to the left.

Reply:
I meant to rename and forgot